.github/workflows/upstream-wpt-changes.yml

```diff
@@ -5,6 +5,8 @@ on:
 
 jobs:
   upstream:
+    # Run job only on servo/servo
+    if: github.repository == 'servo/servo' || github.event_name == 'workflow_dispatch'
```

Comment:
This workflow and test-upstream-wpt-changes are executed in the context of a PR (i.e triggered by pull_request* events), so we can't trigger them manually. The condition on 'workflow_dispatch'  is not going to be useful for these.

Reply:
Good catch. Maybe guard like: `if: github.repository == 'servo/servo' || contains('CI', github.event.pull_request.title)`.

What/How exactly is `upstream-wpt-changes` workflows doing @mrobinson?

Reply:
i answered in #29379 :)